### DISCUSSION TITLE
reverse proxy on a different machine - does not accept connection

### DISCUSSION THREAD

**Discussion by eleaner** [nextcloud/all-in-one#3974] [Questions] [answered]
I set those two variables
```
APACHE_PORT=11000
APACHE_IP_BINDING=0.0.0.0
```
and checked inside of the container
I also forwarded port 11000 (just in case

```
nc -z localhost 11000; echo $? -> 0
nc -z <IP> 11000; echo $? -> 1
```

what I might be doing wrong?

I aim to connect to the AIO from a different server running proxy

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?

**Reply by eleaner** [reply]
I am not sure if the link actually describes the correct procedure - or at least clearly enough

`--env APACHE_PORT=11000` and specifically `nc -z localhost 11000; echo $?` works only when the apache container is started. before there is nothing broadcasting on 11000
The main container still works on 8080, still with self assigned certs etc. and it has to be used to setup the AIO before the 11000 can be made available and reverse proxied to whatever domain.

I was naively expecting that `--env APACHE_PORT=11000` will make AIO setup screens available on 11000
